pkg/helm/watches/watches.go

```diff
@@ -28,28 +28,20 @@ import (
 // Watch defines options for configuring a watch for a Helm-based
 // custom resource.
 type Watch struct {
-	GroupVersionKind        schema.GroupVersionKind
-	ChartDir                string
-	WatchDependentResources bool
-	OverrideValues          map[string]string
+	GroupVersionKind        schema.GroupVersionKind `yaml:",inline"`
+	ChartDir                string                  `yaml:"chart"`
+	WatchDependentResources *bool                   `yaml:"watchDependentResources,omitempty"`
```

Comment:
Pointer?

Reply:
As I understand, while unmarshal and marshal, this will make yaml: "omitempty" effective.